grails-app/assets/javascripts/species.show.js

```diff
@@ -624,7 +626,7 @@ function loadGalleryType(category, start) {
     var url = SHOW_CONF.biocacheServiceUrl  +
         '/occurrences/search.json?q=lsid:' +
         SHOW_CONF.guid +
-        '&fq=multimedia:"Image"&pageSize=' + pageSize +
+        '&qualityProfile=ALA&fq=multimedia:"Image"&pageSize=' + pageSize +
```

Comment:
This should be wrapped in a check for `SHOW_CONF.qualityProfile` so Living Atlas users don't see that param in their web service calls.